packages/components/src/SubHeaderBar/InputTitleSubHeader/TitleSubHeader.component.js

```diff
@@ -47,6 +47,7 @@ function TitleSubHeader({ title, subTitle, onEdit, editable, t }) {
 							theme['tc-subheader-details-text-title-pencil'],
 							'tc-subheader-details-text-title-pencil',
 						)}
+						disabled
```

Comment:
Oops, I think that @jare-talend was right, this syntax is not valid in your case, sorry.